modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js

```diff
@@ -99,18 +100,21 @@ function openToast({
 
 	unmountComponentAtNode(rootElement);
 
-	const onClose = () => unmountComponentAtNode(rootElement);
-
 	const Container =
 		container || containerId ? React.Fragment : ClayAlert.ToastContainer;
 
+	const unmount = () => unmountComponentAtNode(rootElement);
+
+	const onCloseFn = (event) =>
+		onClose ? onClose({event, unmount}) : unmount();
+
 	render(
 		<Container>
 			<ClayAlert
 				autoClose={autoClose}
 				displayType={type}
-				onClick={(event) => onClick({event, onClose})}
-				onClose={onClose}
+				onClick={(event) => onClick({event, onClose: onCloseFn})}
+				onClose={onCloseFn}
```

Comment:
This stands out a bit: `onClick` is defined inline (and calls user-supplied `onClick`) but `onCloseFn` is pulled out into a local var even though it is quite similar (calls user-supplied `onClose`, if provided). I tend to prefer to do similar things in similar ways, so would make this:

```js
	render(
		<Container>
			<ClayAlert
				autoClose={autoClose}
				displayType={type}
				onClick={(event) => onClick({event, onClose: onCloseFn})}
				onClose={(event) =>
					onClose ? onClose({event, unmount}) : unmount()
				}
				title={
					<Text allowHTML={titleType === TYPES.HTML} string={title} />
				}
				variant={variant}
				{...toastProps}
			>
				<Text allowHTML={messageType === TYPES.HTML} string={message} />
			</ClayAlert>
		</Container>,
		renderData,
		rootElement
	);
```

I know the `onClose` attribute is longer than `onClick`, but AFAIK, Prettier isn't even going to wrap it awkwardly.

Reply:
Bah, and now that I actually read it, I see `onCloseFn` used in two places. 😀

Reply:
Yeah, I couldn't think of a better way to share the same `onClose` implementation... which at the same time might be shadowed by the passed `onClose` props... 🤔

Reply:
What about now?